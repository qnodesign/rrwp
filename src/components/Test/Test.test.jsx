import React from 'react';
import Test from './Test';
import { shallow } from 'enzyme';

jest.mock('../../services/Matcher');
jest.mock('../../services/TranslationService');
jest.mock('../../services/RoutingService');

describe('Test', () => {
  test('render', () => {
    const wrapper = shallow(<Test />);
    expect(wrapper).toMatchSnapshot();
  });
});
