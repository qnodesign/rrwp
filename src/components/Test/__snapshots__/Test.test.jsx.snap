// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test is delicious 1`] = `
<div
  className="test"
>
  <h1 />
  <p />
  <button
    onClick={[Function]}
  >
    Male
  </button>
  <button
    onClick={[Function]}
  >
    Female
  </button>
</div>
`;

exports[`Test render 1`] = `
<div
  className="test"
>
  <h1 />
  <p />
  <button
    onClick={[Function]}
  />
  <button
    onClick={[Function]}
  />
</div>
`;
